src/components/dev-hub/card-list.js

```diff
@@ -97,49 +101,72 @@ const renderContentTypeCard = (item, openAudio) => {
     return renderArticle(item);
 };
 
-export default React.memo(({ videos, articles, podcasts, limit = 9 }) => {
-    videos = videos || [];
-    articles = articles || [];
-    podcasts = podcasts || [];
-
-    const fullContentList = sortCardsByDate(videos.concat(articles, podcasts));
-    const [visibleCards, setVisibleCards] = useState(limit);
-
-    const hasMore = fullContentList.length > visibleCards;
-    const [activePodcast, setActivePodcast] = useState(false);
-
-    const openAudio = useCallback(podcast => {
-        setActivePodcast(podcast);
-    }, []);
-    const closeAudio = useCallback(e => {
-        e.stopPropagation();
-        setActivePodcast(null);
-    }, []);
-
-    return (
-        <>
-            <CardContainer data-test="card-list">
-                {fullContentList
-                    .slice(0, visibleCards)
-                    .map(contentType =>
-                        renderContentTypeCard(contentType, openAudio)
-                    )}
-            </CardContainer>
-
-            {hasMore && (
-                <HasMoreButtonContainer>
-                    <Button
-                        secondary
-                        pagination
-                        onClick={() => setVisibleCards(visibleCards + limit)}
-                    >
-                        Load more
-                    </Button>
-                </HasMoreButtonContainer>
-            )}
-            {podcasts.length ? (
-                <Audio onClose={closeAudio} podcast={activePodcast} />
-            ) : null}
-        </>
-    );
-});
+export default React.memo(
+    ({ videos, articles, podcasts, limit = CARD_LIST_LIMIT }) => {
+        const location = useLocation();
```

Comment:
Can simplify to `const { pathname, search } = useLocation()` ?

Reply:
Ah yeah I think I had done this for debugging to look at everything in `location`